### PR TITLE
Import multi 2nd implementation

### PR DESCRIPTION
**Pull request by pedrobranco**
WIP

### CODE REVIEW

File: src/wallet/rpcdump.cpp [hunk 1]
```diff
@@ -615,3 +651,332 @@ UniValue dumpwallet(const UniValue& params, bool fHelp)
     file.close();
     return NullUniValue;
 }
+
+
+UniValue processImportMulti(const UniValue& data) {
```
Comment (promag) [inline src/wallet/rpcdump.cpp]: This is not multi.

File: src/wallet/rpcdump.cpp [hunk 2]
```diff
@@ -615,3 +651,332 @@ UniValue dumpwallet(const UniValue& params, bool fHelp)
...
+    UniValue result = UniValue(UniValue::VOBJ);
```
Comment (promag) [inline src/wallet/rpcdump.cpp]: Not needed so early.

File: src/wallet/rpcdump.cpp [hunk 3]
```diff
@@ -615,3 +651,332 @@ UniValue dumpwallet(const UniValue& params, bool fHelp)
...
+
+    // TODO List:
+    // - Use internal flag.
+    // - Use watchonly flag.
+    // - Check consistency between pubkeys/privkeys and scriptPubKey/redeemScript.
+
+    try {
```
Comment (promag) [inline src/wallet/rpcdump.cpp:662]: Is this here to catch only the following exceptions?

**Comment by pedrobranco** [comment]
replaced by bitcoin's PR.

**Pull request closed without merging**